### DISCUSSION TITLE
Selalu error saat submit capstone di shinny

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#325] [Q&A] [answered]
Selamat siang mas/mbak,

Ini mengapa selalu begini saat submit ? Apakah ada cara lain untuk submit ? 

![image](https://user-images.githubusercontent.com/105972495/191668868-07c5a5e9-45b4-4689-88b1-3d77a24d255b.png)

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Bapak @ChristoNindyo , apakah Bapak sudah memastikan:
1. masuk ke link leaderboard terbaru (via Rpubs): https://algoritma-caps.shinyapps.io/leaderboard
2. memilih case yang tepat pada bagian input
3. memasukan submission file dengan format kolom yang menyesuaikan submission_example.csv 
Apabila problem masih berlanjut, silahkan kirimkan file submission Anda ke mentor@algorit.ma beserta penjelasan problem yang Bapak temui. Terima kasih.
